### ISSUE TITLE
Avoid conditionals GOOD example should be better :)

### ISSUE THREAD

**Issue by FelipeeDev** [jupeter/clean-code-php#84]
As you metion in section: **Open/Closed Principle (OCP)**

You should change the GOOD example in section **Avoid conditionals** like this:

```
interface Airplane
{
    public function getCruisingAltitude();
    // ...
}

class Boeing777 implements Airplane
{
    // ...

    public function getCruisingAltitude()
    {
        return $this->getMaxAltitude() - $this->getPassengerCount();
    }
}

class AirForceOne implements Airplane
{
    // ...

    public function getCruisingAltitude()
    {
        return $this->getMaxAltitude();
    }
}

class Cessna implements Airplane
{
    // ...

    public function getCruisingAltitude()
    {
        return $this->getMaxAltitude() - $this->getFuelExpenditure();
    }
}
```

**Comment by TomasVotruba** [comment]
Hey @FelipeeDev , could you send PR for you improvement?

**Comment by FelipeeDev** [comment]
Sure, already did it. And besides that's you did great work!

**Comment by TomasVotruba** [comment]
Thank you :heart:  I'm just helping @jupeter out to make it better. It's mainly his great work

**Comment by TomasVotruba** [comment]
Resolved by #86
